src/main/java/frc/robot/controls/ManualController.java

```diff
@@ -1,23 +1,30 @@
 package frc.robot.controls;
 
+import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.commands.elevator.MoveToExtension;
+import frc.robot.commands.elevator.ResetEncoderAtBottom;
+import frc.robot.constants.ElevatorConstants;
 import frc.robot.constants.IntakeConstants;
 import frc.robot.constants.OIConstants;
+import frc.robot.subsystems.Elevator;
 import frc.robot.subsystems.FourBarArm;
 import frc.robot.subsystems.Intake;
 import lib.controllers.GameController;
+import lib.controllers.GameController.Axis;
 import lib.controllers.GameController.Button;
 import lib.controllers.GameController.DPad;
 
 public class ManualController {
-  
   GameController m_manual = new GameController(OIConstants.kManualJoy);
   private FourBarArm m_arm;
   private Intake m_intake;
+  private Elevator m_elevator;
   
-  public ManualController(FourBarArm arm, Intake intake){
+  public ManualController(FourBarArm arm, Intake intake, Elevator elevator){
     m_arm = arm;
     m_intake = intake;
+    m_elevator = elevator;
   }
   
```

Comment:
Someone needs to figure out these controls

Reply:
yep. I have some basic ones for now.